misc/config_tools/data/cfl-k700-i7/cfl-k700-i7.xml

```diff
@@ -13,70 +13,67 @@
 	Version: Type2 - Board Version
 	</BASE_BOARD_INFO>
   <PCI_DEVICE>
-	00:00.0 Host bridge: Intel Corporation 8th Gen Core 8-core Desktop Processor Host Bridge/DRAM Registers [Coffee Lake S] (rev 0d)
-	00:02.0 VGA compatible controller: Intel Corporation UHD Graphics 630 (Desktop 9 Series) (rev 02)
+	00:00.0 Host bridge: Intel Corporation Device 3e30 (rev 0d)
```

Comment:
@junjiemao1 do you know why the device ID is listed now instead of the full name? Is it on purpose, or a side-effect of a pic.ids that should be updated?

Reply:
This is simply a dump of the `lspci` output, so any explicit device ID indicates that the ID cannot be found in the pci.ids. possibly due to an outdated pci.ids or the device is not registered even in the latest pci.ids.

Reply:
Thanks for the confirmation @junjiemao1 . Can we update this PR with board XML files that show the full name instead of just the device ID?

Reply:
@jackwhich You may update the pci.ids by executing `sudo update-pciids`.

Reply:
@junjiemao1 oaky